README.md

```diff
@@ -1,4 +1,107 @@
 # OpenFeature project governance
 
-> :warning: Details are coming soon!
+[![Roadmap](https://img.shields.io/static/v1?label=Roadmap&message=public&color=green)](https://github.com/orgs/openfeatureflags/projects/1)
+[![Contributing](https://img.shields.io/static/v1?label=Contributing&message=guide&color=blue)](https://github.com/openfeatureflags/.github/blob/main/CONTRIBUTING.md)
+
+This document describes the bootstrap governance process under which the project will operate
+until the final governance process is identified.
+
+> :warning: This is a temporary (aka bootstrap) governance document that
+> is effective until the project is fully established.
+
+## Project manifesto
+
+### Goal
+
+Our goal is to create an open standard for feature flag management to support a robust feature flag ecosystem using cloud native technologies.
+
+### Scope
+
+The scope includes but not limited to:
+
+- An **open standard** that allows adopters and vendors to easily integrate with a spec-compliant feature flag solutions
+- Creating a **vendor-agnostic ecosystem** supported by the CNCF that provide a seamless experience for developers, SREs, and vendors
+- Unified API and SDK
+- Developer-first, cloud-native reference implementation
+- Extensibility for open source and commercial offerings
+
+### Why OpenFeature?
+
+There are many existing solutions, open source or not.
+Unfortunately, they lack key features needed for wide adoption
+by companies and in the open source ecosystem:
+
+- **Many SDKs** doing the same thing, but slightly differently
+- Many companies **"rebuilding the wheel"**
+- SDKs are generally open source but have no ecosystem
+- **Vendor lock-in** at code-level
+- **No collaboration** across vendors 
+- **Missing integration** with observability tools (e.g. OpenTelemetry)
+
```

Comment:
```suggestion

## CNCF membership

Our goal is to join the [Cloud Native Computing Foundation (CNCF)](https://www.cncf.io/) as a sandbox project shortly after the inception of OpenFeature.
Should it be accepted, the project will establish governance along with the CNCF best practices, 
and then target reaching the incubating and than the graduated statuses in the foundation.

```

Reply:
Seems fine but as I said above I would suggest to define the governance model as early as possible in order to grow a healthy community. It will be easier to convince people to join the project in early days if they see a clear path for them to become an important part of the project (maintainer, governance, technical committee, etc).